### ISSUE TITLE
ColorVariable linter does not report error.

### ISSUE THREAD

**Issue by makotot** [causes/scss-lint#365]
scss-lint(version: 0.33.0) does not report error, though `#ff0` is not in variable declaration.

In .scss-lint.yml
```yml
  ColorVariable:
    enabled: true
```

my test.scss is below.
```scss
.color-variable {
  color: #ff0;
}
```

**Comment by olliebennett** [comment]
Hey @makotot - this new linter is just on master at the moment (it'll be released in v0.34).

For now, try this in your `Gemfile`, and bundle and it should now work! :) (we can update this to 0.34 when it's released).

```ruby
gem 'scss-lint', git: 'https://github.com/causes/scss-lint.git'
```

**Comment by makotot** [comment]
Oops..
@olliebennett Thank you for letting me know.
